cmake/UthequeConfig.cmake.in

```diff
@@ -0,0 +1,30 @@
+# - Config file for the utheque package
+# example:
+# find_package(Utheque REQUIRED)
+# add_executable(your_example example.cpp)
+# target_link_libraries(your_example Utheque)
+
+include(CMakeFindDependencyMacro)
+include(FindPackageHandleStandardArgs)
+
+# CMAKE_MODULE_PATH
+set(CMAKE_MODULE_PATH "@Utheque_CMAKE_MODULE_PATH@")
+
+set(Utheque_INCLUDE_DIRS "@Utheque_INCLUDE_DIRS@")
+
+
+find_package(Boost REQUIRED filesystem)
+
+set(Utheque_LIBRARIES "Threads::Threads;Eigen3::Eigen;Boost::filesystem")
```

Comment:
Do we need `pthread` and `Eigen` for utheque? I would say no.. Just `boost::filesystem`.

Reply:
@jbmouret I already removed those.. Let me know if you think this is an issue..

Reply:
No problem.